brave/browser-laptop
# Add Malay (Malaysia) language

Auditors: @bbondy, @alexwykoff

Test plan:

  1. Select Malay from the language dropdown in about:preferences
  2. Restart browser
  3. Ensure language tokens are in Malay
  4. Select English from the language dropdown in about:preferences
  5. Restart browser
  6. Ensure language can be reset to English
